docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #47.            add_cand_i = not exclude[i] #if it is excluded (i.e. True in <code>exclude</code>), we set add_cand_i=False
</pre><p>I don't like <code>not exclude[i]</code>  as it takes too much mental power to grasp quickly. In hindsight, maybe we want to switch <code>exclude</code>  for <code>include</code>  instead? And then it would be:</p><p> </p><pre class="ql-syntax" spellcheck="false">if include is None:&nbsp;
&nbsp; &nbsp; include = np.ones(n, dtype=bool)
</pre><p> </p><p>and </p><p> </p><pre class="ql-syntax" spellcheck="false">add_cand_i = include[i]&nbsp;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
I got a weird feeling when writing that in the first place. will be fixed!

Reply:
<p>I don't think that's a good idea either. Why not change it to <code>include</code>  everywhere?</p>

Reply:
Since, in `murlin()` function, I have to replace `if np.all(exclude)` with `if np.all(~include)` OR `np.all(include==False)`, I thought it  might be better to just add  `include = ~exclude` here. Maybe I think `if np.all(exclude)` is more readable.

Reply:
<p>IMHO, think <code>np.all(include==False)</code>  is perfectly readable</p>

Reply:
`IMHO` is enough for me :) Will fix it in the next commits!